services/register.js

```diff
@@ -0,0 +1,47 @@
+const AWS = require('aws-sdk');
+AWS.config.update({
+    region: 'us-east-1'
+})
+const utils = require('../utils/utils')
+const bcrypt = require('bcryptjs')
+
+const dynamodb = new AWS.DynamoDB.DocumentClient();
+const userTable = 'jinemister-users';
```

Comment:
I think we need to decide on a userTable here. How does st-employee-db sounds? feel free to suggest your own.

Reply:
It looks good, but don't you think it is a little long. We can use employee-db for this. What do you think?

Reply:
See the closed issue for creating lambda on AWS portal, I have created some resources and we will have to stick to them.

This is the table name: st-employees